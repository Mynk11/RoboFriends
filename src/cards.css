.bor_der {
    border: 1px solid black;
}

.hei_ght {
    height: 200px;
}