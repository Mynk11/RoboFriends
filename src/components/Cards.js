import React from 'react';
import './cards.css'

import 'tachyons';

const cards = ({ id, name, email }) => {
    return (
        <div className="bg-light-green dib bor_der br3 pa3 ma2 grow shadow-5">
            <img src={`https://robohash.org/${name}?200x200`} alt="RoboFriends" className="hei_ght"></img>
            <div className="tc">{id}</div>
            <h2 className="tc">{name}</h2>
            <div className="tc">{email}</div>
        </div>

    );

}

export default cards;