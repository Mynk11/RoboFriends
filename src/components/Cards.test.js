import React from 'react';
import './cards.css'
import { shallow, mount, render } from 'enzyme';
import 'tachyons';
import Cards from './Cards';


console.log(shallow(<Cards />));
it("Should run ", () => {
    //expect(shallow(<Cards />).length).toEqual(1);
    expect(shallow(<Cards />)).toMatchSnapshot();
});
