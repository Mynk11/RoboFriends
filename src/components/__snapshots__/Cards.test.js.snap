// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should run  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <cards />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="RoboFriends"
          className="hei_ght"
          src="https://robohash.org/undefined?200x200"
        />,
        <div
          className="tc"
        />,
        <h2
          className="tc"
        />,
        <div
          className="tc"
        />,
      ],
      "className": "bg-light-green dib bor_der br3 pa3 ma2 grow shadow-5",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "RoboFriends",
          "className": "hei_ght",
          "src": "https://robohash.org/undefined?200x200",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "tc",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "tc",
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "tc",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="RoboFriends"
            className="hei_ght"
            src="https://robohash.org/undefined?200x200"
          />,
          <div
            className="tc"
          />,
          <h2
            className="tc"
          />,
          <div
            className="tc"
          />,
        ],
        "className": "bg-light-green dib bor_der br3 pa3 ma2 grow shadow-5",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "RoboFriends",
            "className": "hei_ght",
            "src": "https://robohash.org/undefined?200x200",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "tc",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "tc",
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "tc",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
